tests/cocotb/test_snitch_cc.py

```diff
@@ -0,0 +1,125 @@
+# ---------------------------------
+# Copyright 2023 KULeuven
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+# Author: Ryan Antonio (ryan.antonio@esat.kuleuven.be)
+# ---------------------------------
+
+
+# -----------------------------------
+# Imports
+# -----------------------------------
+import subprocess
+import cocotb
+from cocotb.triggers import RisingEdge
+from cocotb.clock import Clock
+from cocotb_test.simulator import run
+import pytest
+
+
+# Reconfigurable parameters
+CLOCK_CYCLES = 20
```

Comment:
Many more parameters are available.
We should maybe get  them from a standard `cluster_configuration.hjson` defined somewhere in this repository? This would yield similar behaviour to clustergen.py, but since we can set those parameters here directly, we don't need to use mako for templates (just now?)

Reply:
Ah this parameter does not go into the Snitch core cluster. This is a test parameter. I should replace the comment for that. 

Basically part of this test is just to run a specific number of cycles (for now). It's not part of some `cluster_configuration.hjson` yet. But, yes, I agree we could add something to it later. 

You are correct that we don't need them for now, but we will definitely add the templates later. Because, imagine, for every core we may have different accelerators attached to it and each may have their custom templates. The shell will have a general template but for all other accelerators, we don't know yet. So we need to make a separate set of templates for each core. Not sure how to approach this yet but we'll eventually get there soon.